tools/pubsys/src/aws/ami/wait.rs

```diff
@@ -91,7 +91,8 @@ pub(crate) async fn wait_for_ami(
             successes = 0;
         };
 
-        if attempts % 5 == 1 {
+        // Let the user know we are still trying without being too verbose.
+        if attempts % 10 == 0 && attempts > max_attempts / 2 {
```

Comment:
nit: Maybe it's my personal preference but I like seeing things being outputted to know for sure something is happening and not just hanging. In the PR description error path example, there was nothing outputted for 2 minutes.

```suggestion
        if attempts % 10 == 0 {
```

Reply:
I agree that we should provide some feedback to the user, any time a tool "hangs" without explaining that it is actually doing something is a bad user experience. Ideally we would emit the first time we plan to wait just so the user knows they are entering a waiting period vs "doing things" period in the tool.

Reply:
I reverted this change in https://github.com/bottlerocket-os/bottlerocket/compare/0113236f35b544397d80f315ae54e19831fa5fd1..245eb694562cc69074c6b4039a52a69319b68bcc